tool/teleport/common/teleport.go

```diff
@@ -434,6 +435,15 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 
 	joinOpenSSH.Flag("debug", "Enable verbose logging to stderr.").Short('d').BoolVar(&ccf.Debug)
 
+	integrationCmd := app.Command("integration", "Integration commands")
+	integrationConfigureCmd := integrationCmd.Command("configure", "Configure an integration")
+	integrationConfDeployServiceCmd := integrationConfigureCmd.Command("deployservice-iam", "Create the required IAM Roles for the AWS OIDC Deploy Service.")
+	integrationConfDeployServiceCmd.Flag("cluster", "Teleport Cluster's name.").Required().StringVar(&ccf.IntegrationConfDeployServiceIAMCluster)
+	integrationConfDeployServiceCmd.Flag("integration-name", "Integration name.").Required().StringVar(&ccf.IntegrationConfDeployServiceIAMIntegrationName)
```

Comment:
the integration seems to be superfluous `integration configure --integration-name`  What would you say about just: `integration configure --name foobar`  ?

Reply:
Much better :+1: